src/6_branch.js

```diff
@@ -76,7 +80,6 @@ var wrap = function(parameters, func, init) {
 				else if (parameters === callback_params.CALLBACK_ERR_DATA) {
 					callback(err, data);
 				}
-				next();
```

Comment:
By removing this, if there is no error, next() won't get called. Is it intentional?